packages/sanity/src/core/components/validation/validationList.tsx

```diff
@@ -19,6 +20,7 @@ export interface ValidationListProps {
   truncate?: boolean
 }
 
+/** @internal */
```

Comment:
Cool, I’m not sure of the stability of this one, though. OK to add `@beta` you think?

Reply:
I think this has been stable for a while, so Id say yes. Otherwise keep it hidden, just know that it is used outside (it is in lib in v2 I think, so never sanctioned for use)